<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLTCars</class>
 <widget class="QMainWindow" name="DLTCars">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(148, 186, 255, 255), stop:1 rgba(255, 255, 255, 255));
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="filterButton">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton {
        background-color: #7f04b3;
        border: 2px solid #7f04b3;
        border-radius: 5px;
        color: #FFFFFF;
        padding: 4px 12px;
        font-size: 12px;
    }
    
    QPushButton:hover {
        background-color: #0056b3;
        border-color: #0056b3;
    }
    
    QPushButton:pressed {
        background-color: #004080;
        border-color: #004080;
    }</string>
      </property>
      <property name="text">
       <string>filterButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="3">
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
        background-color: #FFFFFF;
        border: 1px solid #C0C0C0;
        border-radius: 5px;
        font-family: Arial;
        gridline-color: #C0C0C0;
    }
    
    QTableWidget::item {
        padding: 4px;
    }
    
    QTableWidget::item:selected {
        background-color: #E6F7FF;
        color: #000000;
    }
    
    QTableWidget::item:hover {
        background-color: #F0F0F0;
    }
    
    QHeaderView::section {
        background-color: #F0F0F0;
        border: none;
        font-size: 12px;
        font-weight: bold;
    }
    
    QHeaderView::section:checked {
        background-color: #E6F7FF;
        color: #000000;
    }
    
    QTableCornerButton::section {
        background-color: #F0F0F0;
        border: none;
    }</string>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>80</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QTableWidget" name="filteredTableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>600</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
        background-color: #FFFFFF;
        border: 1px solid #C0C0C0;
        border-radius: 5px;
        font-family: Arial;
        gridline-color: #C0C0C0;
    }
    
    QTableWidget::item {
        padding: 4px;
    }
    
    QTableWidget::item:selected {
        background-color: #E6F7FF;
        color: #000000;
    }
    
    QTableWidget::item:hover {
        background-color: #F0F0F0;
    }
    
    QHeaderView::section {
        background-color: #F0F0F0;
        border: none;
        font-size: 12px;
        font-weight: bold;
    }
    
    QHeaderView::section:checked {
        background-color: #E6F7FF;
        color: #000000;
    }
    
    QTableCornerButton::section {
        background-color: #F0F0F0;
        border: none;
    }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="autoScrollMargin">
       <number>3</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>10</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>10</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QListView" name="listView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTreeView" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
QTreeViex{
background-color:#ffffff;
color:#026E81;}</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="filterLineEdit">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
        background-color: #FFFFFF;
        border: 1px solid #C0C0C0;
        border-radius: 5px;
        padding: 4px;
        font-family: Arial;
        font-size: 14px;
    }

    QLineEdit:focus {
        border-color: #007BFF;
        outline: none;
    }

    QLineEdit:hover {
        background-color: #F0F0F0;
    }</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>277</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QMenuBar {
        background-color: #94baff;
        color: #FFFFFF;
        border: none;
        font-family: Arial;
        font-size: 9px;
    }

    QMenuBar::item {
        background-color: transparent;
        padding: 5px 10px;
    }

    QMenuBar::item:selected {
        background-color: #1DA1F2;
    }

    QMenu {
        background-color: #292F33;
        border: 1px solid #1DA1F2;
    }

    QMenu::item {
        color: #FFFFFF;
        padding: 6px 20px;
        background-color: transparent;
    }

    QMenu::item:selected {
        background-color: #1DA1F2;
    }</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter </string>
    </property>
    <addaction name="actionAdd"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar {
	background-color: qradialgradient(spread:reflect, cx:0.500148, cy:0.505, radius:2, fx:0.5, fy:0.5, stop:1 rgba(255, 255, 255, 45));

   
        border: none;
        font-family: Arial;
    }

    QToolBar QToolButton {
        background-color: transparent;
        border: none;
        color: #000000;
        padding: 2px 3px;
        font-size: 12px;
    }

    QToolBar QToolButton:hover {
        background-color: #E6E6E6;
    }

    QToolBar QToolButton:checked {
        background-color: #D9D9D9;
    }</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
